scripts/lint-style.py

```diff
@@ -149,6 +152,45 @@ def long_lines_check(lines, path):
             errors += [(ERR_LIN, line_nr, path)]
     return errors
 
+def indent_check(lines, path):
+    errors = []
+    indent_lvl = 0
+    in_prf = 0 # counter for nested proof blocks
+    check_rest_of_block = True # we only check uncomplicated syntax
+    ended_with_comma = False # track whether the previous line ends with a comma
+    inside_special = 0 # track whether we are inside ⟨⟩ or []
+    for line_nr, line in enumerate(lines, 1):
+        lstr = line.lstrip(' ') # strip spaces from beginning of line
```

Comment:
nit: PEP8 wants two spaces before `#`:
```suggestion
    in_prf = 0  # counter for nested proof blocks
    check_rest_of_block = True  # we only check uncomplicated syntax
    ended_with_comma = False  # track whether the previous line ends with a comma
    inside_special = 0  # track whether we are inside ⟨⟩ or []
    for line_nr, line in enumerate(lines, 1):
        lstr = line.lstrip(' ') # strip spaces from beginning of line
```